queries/places.jinja2

```diff
@@ -38,7 +38,7 @@ SELECT
     NULL AS labelrank,
     tags->'population' AS population,
     tags->'capital' AS capital,
```

Comment:
`country_capital` would require another change in the YAML file for Natural Earth.